backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/metadata.js

```diff
@@ -17,37 +17,46 @@
 
 const { createResponseCode, templates } = require('../../../../../../utils/responseCodes');
 const Yup = require('yup');
-const { getMetadataById } = require('../../../../../../models/metadata');
+const { getMetadataByQuery } = require('../../../../../../models/metadata');
 const { respond } = require('../../../../../../utils/responder');
 
 const Metadata = {};
 
 const metadataSchema = Yup.object().shape({
-	key: Yup.string().min(1).max(120).required(),	
+	key: Yup.string().min(1).max(120).required(),
 	value: Yup.lazy((value) => {
 		switch (typeof value) {
-			case 'string':
-				return Yup.string().max(120);
-			case 'number':
-				return Yup.number();
-			case 'boolean':
-				return Yup.bool();			  
-			default:
-				return Yup.mixed().nullable(true).test('ensure-value-present', 'Metadata value is a required field', () => value !== undefined);
-		  }
-	})
+		case 'string':
```

Comment:
Default being mixed doesn't make too much sense, if it's not a number or boolean, we want to make sure it's a string (or at least can be casted into a string)

```suggestion
		case 'number':
			return Yup.number();
		case 'boolean':
			return Yup.bool();
		default:
			return YupHelper.types.strings.title().nullable() //or whatever tests you need

```